src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -0,0 +1,53 @@
+import clsx from 'clsx';
+import React, { FC, useState } from 'react';
+import { HexColorPicker } from 'react-colorful';
+
+import './ColorPicker.scss';
+import { ColorPickerProps } from './ColorPicker.types';
+
+const ColorPicker: FC<ColorPickerProps> = ({
+	color,
+	onChange,
+	disabled = false,
+	className,
+	rootClassName: root = 'c-color-picker',
+	style,
+}) => {
+	const [showColorPicker, setShowColorPicker] = useState<boolean>(false);
+
+	return (
+		<div
+			className={clsx(root, className, { 'c-color-picker__disabled': disabled })}
+			style={style}
+		>
+			<div className="c-color-picker__controls">
+				<button
+					className="c-color-picker__trigger"
+					style={{ backgroundColor: color }}
+					onClick={() => setShowColorPicker((show) => !show)}
+					{...(disabled ? { disabled } : {})}
```

Comment:
Wrm niet gwn `disabled={disabled}` ?

Reply:
omdat de input field enkel kijkt naar het attribute, niet de value.
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled

Dit resulteert in een disabled input field. Tenzij React dat op een of andere manier opvangt?
```
<input disabled="false"/>
```

Reply:
React vangt die boolean value op voor disabled attr om compliant te zijn met de html spec